aloscene/tensors/augmented_tensor.py

```diff
@@ -48,34 +48,29 @@ def __init__(self, x, **kwargs):
         super().__init__()
 
     def drop_children(self):
-        """ Remove all children from this augmented tensor and return
+        """Remove all children from this augmented tensor and return
         the removed children.
         """
         labels = {}
         for name in self._children_list:
-            labels[name] = {
-                "value": getattr(self, name),
-                "property": self._child_property[name]
-            }
+            labels[name] = {"value": getattr(self, name), "property": self._child_property[name]}
```

Comment:
your ide might not be configured properly

Reply:
I have same configuration as the one which is recommanded in the aloception gitlab.